helm_deploy/values-dev.yaml

```diff
@@ -1,34 +1,16 @@
 
-minReplicaCount: 1
-maxReplicaCount: 2
+  ingress:
+    hosts:
+      - crime-portal-gateway-dev.apps.live-1.cloud-platform.service.justice.gov.uk
 
-image:
-  repository: quay.io/hmpps/crime-portal-gateway
-  tag: latest
-  port: 8081
+  env:
+    SPRING_PROFILES_ACTIVE: "dev,logstash"
+    INCLUDED_COURT_CODES: "B01BE,B01BH,B01CE,B01CN,B01DU,B01EF,B01FA,B01GQ,B01GU,B01IX,B01KR,B01LA,B01LY,B01MN,B01ND,B01NM,B01OK,B03AX,B03DE,B03OT,B04BP,B04BQ,B04CO,B04HQ,B04KO,B05BK,B05BW,B05PK,B06AN,B06BV,B06IS,B06MK,B06OJ,B07DM,B07ED,B07FQ,B10BD,B10BF,B10FR,B10JL,B10JQ,B10LX,B10MR,B11EI,B11JP,B11KF,B12GH,B12LK,B12LT,B12PA,B13CC,B13HD,B13HT,B14AV,B14ET,B14LO,B16BG,B16CJ,B16GB,B16HE,B17JA,B20BL,B20EB,B20EY,B20NQ,B20OQ,B21DA,B21JI,B22GR,B22HM,B22KS,B22MZ,B22OS,B22OS,B23HS,B23HS,B23PP,B30PG,B30PG,B30PI,B31IT,B31JV,B32BX,B32HX,B33HU,B33II,B34JS,B34NX,B35CZ,B35HF,B35KE,B36FZ,B36HN,B36JU,B37HI,B37HI,B40IM,B41ME,B41MJ,B41US,B42AZ,B42AZ,B42CM,B42CM,B42CO,B42CO,B42MB,B42MB,B43JC,B43KB,B43KQ,B43LV,B43OX,B44AG,B44BA,B44JK,B44KM,B44MA,B45GC,B45MH,B46DB,B46DH,B46FO,B46IR,B46IU,B46LN,B47CL,B47EC,B47GL,B47HB,B47OV,B47OV,B50AW,B50BY,B50VR,B50JO,B50KH,B50NL,B52BB,B52CM,B52MY,B52OC,B52OZ,B53DJ,B54MW,B54WV,B55KL,B55OE,B60ID,B60JE,B60OW,B60WU,B61EH,B61NP,B62DC,B62IZ,B62KL,B62MV,B63AD,B63GN,B63IC,B63IE,B63NZ"
 
-ingress:
-  enabled: true
-  enable_whitelist: true
-  hosts:
-    - host: crime-portal-gateway-dev.apps.live-1.cloud-platform.service.justice.gov.uk
-  path: /mirrorgateway/service/cpmgwextdocapi(/|$)(.*)
-  rewrite_target: /crime-portal-gateway/ws/$2
-
-env:
-  JAVA_OPTS: "-Xmx512m"
-  SPRING_PROFILES_ACTIVE: "dev,logstash"
-  INCLUDED_COURT_CODES: "B01BE,B01BH,B01CE,B01CN,B01DU,B01EF,B01FA,B01GQ,B01GU,B01IX,B01KR,B01LA,B01LY,B01MN,B01ND,B01NM,B01OK,B03AX,B03DE,B03OT,B04BP,B04BQ,B04CO,B04HQ,B04KO,B05BK,B05BW,B05PK,B06AN,B06BV,B06IS,B06MK,B06OJ,B07DM,B07ED,B07FQ,B10BD,B10BF,B10FR,B10JL,B10JQ,B10LX,B10MR,B11EI,B11JP,B11KF,B12GH,B12LK,B12LT,B12PA,B13CC,B13HD,B13HT,B14AV,B14ET,B14LO,B16BG,B16CJ,B16GB,B16HE,B17JA,B20BL,B20EB,B20EY,B20NQ,B20OQ,B21DA,B21JI,B22GR,B22HM,B22KS,B22MZ,B22OS,B22OS,B23HS,B23HS,B23PP,B30PG,B30PG,B30PI,B31IT,B31JV,B32BX,B32HX,B33HU,B33II,B34JS,B34NX,B35CZ,B35HF,B35KE,B36FZ,B36HN,B36JU,B37HI,B37HI,B40IM,B41ME,B41MJ,B41US,B42AZ,B42AZ,B42CM,B42CM,B42CO,B42CO,B42MB,B42MB,B43JC,B43KB,B43KQ,B43LV,B43OX,B44AG,B44BA,B44JK,B44KM,B44MA,B45GC,B45MH,B46DB,B46DH,B46FO,B46IR,B46IU,B46LN,B47CL,B47EC,B47GL,B47HB,B47OV,B47OV,B50AW,B50BY,B50VR,B50JO,B50KH,B50NL,B52BB,B52CM,B52MY,B52OC,B52OZ,B53DJ,B54MW,B54WV,B55KL,B55OE,B60ID,B60JE,B60OW,B60WU,B61EH,B61NP,B62DC,B62IZ,B62KL,B62MV,B63AD,B63GN,B63IC,B63IE,B63NZ"
-
-whitelist:
-  office: "217.33.148.210/32"
-  health-kick: "35.177.252.195/32"
-  mojvpn: "81.134.202.29/32"
-  cloudplatform-live1-1: "35.178.209.113/32"
-  cloudplatform-live1-2: "3.8.51.207/32"
-  cloudplatform-live1-3: "35.177.252.54/32"
-  bsi-1: "54.37.241.156/30"
-  bsi-2: "167.71.136.237/32"
-  global-protect: "35.176.93.186/32"
+  allowlist:
```

Comment:
i'm guessing if we want to send a payload to dev CPG, we'll just use the svc url e.g. https://serviceurl.com/

Reply:
The `internal` range should mean we have the same access as we do now